featureless-void.cabal

```diff
@@ -23,6 +23,7 @@ library
                      Helper
                      Helper.S3
                      Helper.Time
+                     Helper.Twitter.Types
```

Comment:
I get into a bad habit of throwing everything in "Helpers" as well but I think it's a smell. This could probably be `Types.Twitter`?